.github/workflows/release.yml

```diff
@@ -571,6 +582,20 @@ jobs:
               process.exit(1);
             });
 
+            // website changelog
+            const changelogText = getWebsiteChangelog({ version, issues });
+
+            await github.rest.repos.createDispatchEvent({
+              owner: context.repo.owner,
+              repo: '${{ vars.WEBSITE_REPO }}',
+              event_type: 'update-changelog',
+              client_payload: {
+                version: getGenericVersion(version),
+                majorVersion: getMajorVersion(version),
+                text: changelogText,
+              }
+            }).catch(console.warn);
```

Comment:
this failing for any reason should not fail the job, just print out an error message.